src/OpenTK/Platform/MacOS/HIDInput.cs

```diff
@@ -850,8 +850,8 @@ static void UpdateJoystick(JoystickData joy, IOHIDValueRef val)
                         case HIDUsageGD.Dial:
                         case HIDUsageGD.Wheel:
                             short offset = GetJoystickAxis(val, elem);
-                            JoystickAxis axis = JoystickAxis.Axis0 + joy.Elements[cookie].Index;
-                            if (axis >= JoystickAxis.Axis0 && axis <= JoystickAxis.Last)
+                            int axis = joy.Elements[cookie].Index;
+                            if (axis >= 0 && axis <= 64)
```

Comment:
Use the MaxAxes constant instead of an inlined value in case it changes later on.